test/infisical/scenarios.json

```
@@ -0,0 +1,17 @@
+{
+  "test_debian": {
+    "image": "mcr.microsoft.com/devcontainers/base:debian",
+    "features": {
+      "infisical": {}
+    }
+  },
+  "test_specific_version": {
+    "image": "mcr.microsoft.com/devcontainers/base:debian",
+    "features": {
+      "infisical": {
+        "version": "0.43.46"
```

```suggestion
        "version": "0.43.45"
```

Let's not use here the latest version here